sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -237,7 +237,444 @@
       <DependentUpon>SomeResources.resx</DependentUpon>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Compile>
-    <Compile Include="**\*.cs" Exclude="**\obj\**\*;**\bin\**\*;**\*.Designer.cs;TestCases\**\*.cs;TestCasesForRuleFailure\**\*.*" />
+    <Compile Include="Common\AnalyzerAdditionalFileTest.cs" />

```

Comment:
Did you intend to keep this change to the project file?

Reply:
no....